packages/slice-machine/src/modules/selectedCustomType/selectors.ts

```diff
@@ -34,9 +34,7 @@ export const selectCurrentPoolOfFields = (
   );
 };
 
-export const selectIsCurrentCustomTypeHasPendingModifications = (
-  store: SliceMachineStoreType
-) => {
+export const isSelectedCustomTypeTouched = (store: SliceMachineStoreType) => {
```

Comment:
Should we still use Touch of modified ?

Reply:
wdym?

Reply:
Modified is word used for statuses, it has a meaning in the APP and for us developpers.
I was just wondering if `Touched` would be as clear for developers.
It seems very frenchy that's why i'm wondering :)